flimfit.html

```diff
@@ -0,0 +1,880 @@
+
+<ol class="step">
+
+<li><p class="first">Back in FLIMfit, click on the <span class="bold">OMERO</span> menu and select <span class="bold">Load FLIM data from a dataset</span>.</p>
+
+<p>The OMERO data tree will be displayed in a pop-up window.</p>
+
+<p>Select the <span class="code-example">Multiple</span> dataset and click <span class="bold">Open</span>.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/flimfitMultiOpen.jpg" class="align-center" src="images/flimfitMultiOpen.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li><p class="first">If loading a file from disk, click on the <span class="bold">File</span> menu.</p>
+
+<p>Select <span class="bold">Load FLIM Dataset</span> and select <span class="bold">Load FLIM Dataset...</span>.</p>
+
+<p>In the dialog box, select the folder: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">Multiple</span>.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/flimfitMultiOpenDisk.jpg" class="align-center" src="images/flimfitMultiOpenDisk.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">To load the IRF for the multiple files from the attachment on the dataset, select the <span class="bold">OMERO</span> menu and choose <span class="bold">Load IRF (Annotation)</span>.</p>
+
+<p>In the dialog box, select the folder: &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<span class="code-example">Multiple</span>.</p>
+
+<p>Click <span class="bold">Open</span> and then click <span class="bold">OK</span>.</p>
+
+<p>The IRF data will be visible as a red peak on the plot in the <span class="bold">Decay</span> tab.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/flimfitMultiIDR.jpg" class="align-center" src="images/flimfitMultiIDR.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li><p class="first">Set the <span class="bold">Integrated Min</span> to 90 to exclude the dark areas of the image.</p>
+
+<p>Select <span class="bold">2</span> in the <span class="bold">No Expo</span> drop-down.</p>
+
+<p>Click <span class="bold">Fit Dataset</span>.</p>
+
```

Comment:
In the pdf, the line 'A broken red line appears in the fitted model.' is added here.